contrib/utilities/indent

```diff
@@ -29,14 +29,13 @@
 #   make indent
 # from a build directory.
 #
-
 if test ! -d source -o ! -d include -o ! -d examples ; then
   echo "*** This script must be run from the top-level directory of deal.II."
   exit 1
 fi
 
 # Add the location 'setup_astyle.sh' installs astyle to to the local PATH.
```

Comment:
@luca-heltai maybe add comment that we don't want to use `realpath` as it does not exist on all platforms.